### PR TITLE
Python 3

### LINKED ISSUE #1: Python 3
Need to enable Python 3 builds once ecmwf_grib is available on py3k. (conda-forge/ecmwf_grib-feedstock#1)

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,10 +8,13 @@ source:
   fn: pygrib-{{ version }}.tar.gz
   url: https://pypi.python.org/packages/source/p/pygrib/pygrib-{{ version }}.tar.gz
   md5: e9ae04cb987992691b388b16be53214a
+  patches:
+      # Fixed tab/space inconsistency.
+      - tab_space.patch
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I am submitting this upstream, but we don't have to wait for a new release to get `pygrib` working on Python 3.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -43,3 +46,4 @@ about:
 extra:
   recipe-maintainers:
     - dopplershift
+    - jjhelmus
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Ping @jjhelmus (See https://github.com/conda-forge/staged-recipes/pull/372/files/0e1cc6aee8f77b9cc5539d8561bc9ed9280b12b6#r60065433)

**Comment by jjhelmus** [comment]
LGTM, is this ready to merge @ocefpaf?

**Comment by ocefpaf** [comment]
> LGTM, is this ready to merge @ocefpaf?

:+1:

**Comment by dopplershift** [comment]
🎉

**Pull request merged**